pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/NonPersistentTopics.java

```diff
@@ -541,7 +541,7 @@ public void getEntryFilters(@Suspended AsyncResponse asyncResponse,
     @Path("/{tenant}/{namespace}/{topic}/entryFilters")
     @ApiOperation(value = "Set entry filters for specified topic")
     @ApiResponses(value = {@ApiResponse(code = 403, message = "Don't have admin permission"),
-            @ApiResponse(code = 404, message = "Topic does not exist"),
+            @ApiResponse(code = 404, message = "Tenant or namespace or topic doesn't exist"),
```

Comment:
Is there a reason this string has `Tenant`, but most of the others do not?

Reply:
Already fixed. Could you take a look again?

Reply:
I don't think this was fixed. Why do we have `tenant` in this annotation but not in the annotation on line 192 above?